store/blocks.json

```diff
@@ -92,6 +93,20 @@
       }
     }
   },
+  "info-card": {
+    "props": {
+      "isFullModeStyle": true,
+      "textPosition": "left",
+      "headline": "Enjoy the outside with style",
+      "subhead": "Choose from a selection of our best Panama Hats",
+      "callToActionMode": "button",
+      "callToActionText": "Shop Panama Hats",
+      "callToActionUrl": "hats/panama",
+      "imageUrl": "https://i.ibb.co/qYktksq/resized.png",
```

Comment:
🤔 how about hosting this on vtex servers?

Reply:
will take a look on how to do this